pkg/specgen/generate/container_create.go

```diff
@@ -128,7 +130,39 @@ func createContainerOptions(ctx context.Context, rt *libpod.Runtime, s *specgen.
 	if s.Stdin {
 		options = append(options, libpod.WithStdin())
 	}
-	if len(s.Systemd) > 0 {
+
+	useSystemd := false
+	switch s.Systemd {
+	case "always", "true":
```

Comment:
I think `"true"` should be handled as `""`, not `"always"`, that is the behaviour in 1.9